plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -1039,7 +1039,7 @@ writeDataError(wrkrInstanceData_t *pWrkrData, instanceData *pData, fjson_object
 			DBGPRINTF("omelasticsearch: error creating file content.\n");
 			ABORT_FINALIZE(RS_RET_ERR);
 		}
-		rendered = (char*)fjson_object_to_json_string(ctx.errRoot);
+		rendered = strdup((char*)fjson_object_to_json_string(ctx.errRoot));
```

Comment:
This is sub-optimal from a performance point of view; however, we should not run into this situation too frequently. So IMHO code simplicity wins (as suggested).